packages/framework/presence/src/test/presenceManager.spec.ts

```diff
@@ -90,8 +90,8 @@ describe("Presence", () => {
 			});
 
 			describe("attendee", () => {
-				const newAttendeeSessionId = "sessionId-4";
-				const initialAttendeeConnectionId = "client4";
+				const newAttendeeSessionId: string = "sessionId-4";
+				const initialAttendeeConnectionId: string = "client4";
```

Comment:
?? mistake?

Reply:
Removed, I was previously getting phantom eslint errors here

Reply:
I see a lot of those in VS Code. reloading the window usually helps. When I am suspicious I will rely on command line for accuracy.